clients/client-java/src/main/java/org/apache/gravitino/client/GravitinoMetalake.java

```diff
@@ -762,6 +771,55 @@ public Group revokeRolesFromGroup(List<String> roles, String group)
     return resp.getGroup();
   }
 
+  /**
+   * Get the owner of a metadata object.
+   *
+   * @param object The metadata object
+   * @return The owner of the metadata object. If the metadata object doesn't set the owner, it will
+   *     return Optional.empty().
+   * @throws NotFoundException If the metadata object is not found.
+   */
+  public Optional<Owner> getOwner(MetadataObject object) throws NotFoundException {
```

Comment:
I can't found anywhere throws NotFoundException, and when I deleted `throws NotFoundException`  we can also compile success.

Reply:
Because `NotFoundException` is a RuntimeException. So it compile successfully. It throws in the ErrorHandler.